src/main/java/io/codelex/oop/parcels/Parcel.java

```diff
@@ -0,0 +1,38 @@
+package io.codelex.oop.parcels;
+
+public class Parcel implements Validatable {
+
+    private int xLength;
+    private int yLength;
+    private int zLength;
+    private float weight;
+    private boolean isExpress;
+
+    public Parcel(int xLength, int yLength, int zLength, float weight) {
+        this.xLength = xLength;
+        this.yLength = yLength;
+        this.zLength = zLength;
+        this.weight = weight;
+    }
+
+    @Override
+    public boolean validate() {
+        float MAX_WEIGHT = 30.0f;
```

Comment:
mierīgi klases līmenī kā konstanti, caps lock only for static

Reply:
Izlabots